src/plugins/show-tips/tips.ts

```diff
@@ -212,6 +16,49 @@ function hashString(str: string) {
   );
 }
 
-tips.sort((a, b) => hashString(a.desc) - hashString(b.desc));
-
-export default tips;
+export function getTipData(): {
+  tipKeys: string[];
+  learnMore: Record<string, string | undefined>;
+} {
+  return {
+    tipKeys: getMessageNames(/^show-tips-tip/g).sort(
+      (a, b) => hashString(a) - hashString(b)
+    ),
+    learnMore: {
+      "show-tips-tip-arctan": "https://www.desmos.com/calculator/beqk9rnxbl",
+      "show-tips-tip-indefinite-integral": `https://help.desmos.com/hc/${currentLanguage()}/articles/4406810279693-Integrals`,
```

Comment:
These articles don't have translations. Use the `en-us` link for all of these.

Reply:
fixed!